src/components/Global/RadioButton/index.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import styles from './RadioButton.module.scss';
+
+type DIFFICULTY = 'easy' | 'medium' | 'hard';
+
+type RadioButtonProps = {
+  id: string /*radioEasy*/;
+  name: string /*filter*/;
+  difficulty: DIFFICULTY;
+  checked: boolean;
+  setChecked: React.Dispatch<React.SetStateAction<boolean>>;
```

Comment:
Oh is this the proper way to do this? It looks kinda clunky

Reply:
yee. The old one was wrong and more clunky.